corehq/apps/app_manager/tests/data/one_question_two_images.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
It's odd that a test in hqmedia pulls its data from app_manager. This whole feature straddles hqmedia and app_manager. I'm putting most of the code in hqmedia largely because it's a smaller app.